internals/overlord/servstate/manager.go

```diff
@@ -30,7 +34,8 @@ type ServiceManager struct {
 	randLock sync.Mutex
 	rand     *rand.Rand
 
-	logMgr LogManager
+	workloads map[string]workload.Workload
```

Comment:
Since workloads are the mechanism which will evolve over time to provide service level access permissions and confinement features, for me it makes sense that the Service Manager owns the configuration.